LadybugTools_oM/Collections/Collection/DoublesCollection.cs

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Linq;
+using BH.oM.Base;
+
+namespace BH.oM.LadybugTools
+{
+    public class DoublesCollection : BHoMObject, ICollection
+    {
+        [Description("A list of doubles.")]
+        public virtual List<double> Values { get; set; } = null;
```

Comment:
Why is this here? This hardly seems beneficial to the oM? Is this meant to help the adapter work? If so, this should go into an `Objects` folder within the adapter rather than the public oM.

Reply:
After discussion, `HourlyContinuousCollection` will use List<string> for collections, which while it means that anyone using the lists as doubles must parse the strings to doubles, it also removes the Doubles/Strings Collection objects so users will be less confused.